aea/helpers/async_utils.py

```diff
@@ -61,6 +64,9 @@ def ensure_list(value: Any) -> List:
     return [value]
 
 
+not_set = object()
```

Comment:
Could you explain to me why this can't be just `None`? I might miss something.

Reply:
cause None is the valid value for the state, and when we want check for value is not defined bu can be set to None we should use something like NonSet value